report_thesis/src/sections/background/linear_and_regularization_models/pls.tex

```diff
@@ -1,46 +1,23 @@
-
-One drawback of \gls{pcr} is that it does not consider the target in the decomposition of the features and therefore assumes that smaller components have a weaker correlation with the target than the larger ones.
-This assumption does not always hold, which is what \gls{pls} aims to address.
+One drawback of \gls{pcr} is that it does not consider the target variable in the decomposition of the features and therefore assumes that components with larger variance have a stronger correlation with the target than those with smaller variance. In other words, the components that capture the most variation may not be the most predictive of the target. Some data might be highly variable but irrelevant to the target.
 
+To address this limitation, \gls{pls} extends \gls{pcr} by considering the target variable when identifying the components.
 \gls{pls} uses an iterative method to identify components that maximize the covariance between the features and the target.
-These components, $Z$, are linear combinations of the original features, $\mathbf{X}_j$, weighted by coefficients, $\phi_j$, which are specifically calculated to reflect this covariance.
-The formula for each component is expressed as:
-
-$$
-    Z = \sum_{j=1}^{p} \phi_j \mathbf{X}_j,
-$$
-
-where $Z$ represents the component, $\mathbf{X}_j$ is the $j$-th feature, and $\phi_j$ is the weight for the $j$-th feature.
-The weights, $\phi_j$, are determined by the formula:
-
-$$
-    \phi_j = \frac{\text{cov}(\mathbf{X}_j, Y)}{\text{var}(\mathbf{X}_j)}.
-$$
-
-To refine the model iteratively, PLS uses the residuals from the previous components to calculate the next component.
-The $m$-th component, for example, is derived from the residuals of the previous $m-1$ components:
-
-$$
-    Z_m = \sum_{j=1}^{p} \phi_{jm} \hat{\mathbf{X}}_{j, m-1}.
-$$
-
-The components are then used to predict the target variable by fitting a linear model via least squares regression.
\ No newline at end of file
+Similar to \gls{pcr}, \gls{pls} identifies $k$ components that capture the most variance in the data and fits a linear model with least squares regression.
+However, unlike \gls{pcr}, \gls{pls} uses the target variable to identify the components that are most predictive of the target, resulting in components that not only approximate the data but also relate to the target variable.
+In essence, \gls{pls} aims to find components that are both informative about the data and predictive of the target.
+
+This is an iterative process where the residuals from the previous components are used to calculate the next component.
+The $m$-th component, for example, is derived from the residuals of the previous $m-1$ components.
+One can interpret these residuals as the part of the data that has not been explained by the previous components.
+After some number of iterations, the components are used to predict the target variable by fitting a linear model via least squares regression, just like in \gls{pcr}.
```

Comment:
I think a key aspect missing here is that the number of iterations done is equal to the number of components chosen.
In the book they write:
"This iterative approach can be repeated M times to identify multiple PLS components Z1,...,ZM" So you have as many PLS components as you have M.

Reply:
Should be fixed now.